### PR TITLE
Adding Submitted Bios

### PR DESCRIPTION
**Pull request by BenWoodford76**
Adding bios submitted from the person collect form

### CODE REVIEW

File: _committees/07_08.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ committee:
     name: Maia Gibbs
   - role: External Events Coordinator
     name: Sam Turnbull
+    note: Stepped down during the year
+  - role: External Events Coordinator
+    name: Simon Thompson
+    note: Took over after Sam Turnbull stepped down
```
Comment (samozzy) [inline _committees/07_08.md:30]: Can we add a `(dates unknown)` to these as a mini call to action for more info? Thoughts?

File: _people/annabel_byrom.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+submitted: 2020-11-14 
+gender: female  
+course: 
+  - Philosophy, Politics and Economics
+graduated: 1997
+contact_allowed: true
+
+--- 
+
+<!-- Content for a bio here -->
```
Comment (samozzy) [inline _people/annabel_byrom.md:16]: This line can go away

File: _people/john_schwiller.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - type: Instagram
+    username: iJohnDNS
+  - type: LinkedIn
+    username: john-schwiller-99296
+
+--- 
+
+After leaving uni in 76 I went into rep at Leatherhead as an electrician/Lighting Designer, continuing the pyromania on the panto and almost burning down the theatre.
+
+I then joined CCT Theatre Lighting where I spent 10yrs as Technical Director, designing lighting equipment as seen in Abbas Super Trouper, Human Leagues Dont You Want Me, Cats, a Joe Jackson tour and much of the first fit out of the National Theatre.
```
Comment (samozzy) [inline _people/john_schwiller.md:25]: Can we get some apostrophes in here please?

File: _shows/05_06/the_slippery_soapbox_edinburgh.md [hunk 4]
```diff
@@ -13,6 +13,10 @@ assets:
   - type: poster
     image: bQg3F6W
 
+cast:
+  - role: Graham/Leotard Man
```
Comment (samozzy) [inline _shows/05_06/the_slippery_soapbox_edinburgh.md:17]: Spaces around the `/` please

**Review by samozzy — CHANGES_REQUESTED** [review]

File: _shows/01_02/cabaret.md [hunk 5]
```diff
@@ -11,5 +11,8 @@ date_end: 2001-12-08
 crew:
  - role: Musical Director
    name: Robert Wicks
+ - role: Band, Trombone
```
Comment (samozzy) [inline _shows/01_02/cabaret.md:14]: Can this be
```
role: Band
note: Trombone
```
Which I think then matches up with our other formats, as I don't think there's a programme formatted in that way for this show.

**Comment by BenWoodford76** [comment]
@samozzy think i've addressed everything :)

**Review by samozzy — APPROVED** [review]

**Review by samozzy — APPROVED** [review]

**Pull request merged**